app/(site)/about/MenuDescription.tsx

```diff
@@ -0,0 +1,190 @@
+      },
+      {
+        title: "Cultural",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+    ],
+  },
+  {
+    title: "Desporto e Bem-Estar",
+    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    avatars: [<Avatar key="7" />, <Avatar key="8" />, <Avatar key="9" />],
+    subgroups: [
+      {
+        title: "Desportivo",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+      {
+        title: "Saúde e Sustentabilidade",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+    ],
+  },
+  {
+    title: "Divulgação",
+    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    avatars: [<Avatar key="10" />, <Avatar key="11" />, <Avatar key="12" />],
+    subgroups: [
+      {
+        title: "Design",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+      {
+        title: "Comunicação e Marketing",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+      {
+        title: "Vídeo",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+    ],
+  },
+  {
+    title: "Política Educativa",
+    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    avatars: [<Avatar key="13" />, <Avatar key="14" />, <Avatar key="15" />],
+    subgroups: [
+      {
+        title: "Cursos",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+      {
+        title: "Workshops",
+        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      },
+    ],
+  },
+];
+
+const MenuDescription = () => {
+  const [selected, setSelected] = useState<number | null>(null);
+  const [menuHeight, setMenuHeight] = useState(0);
+  const menuRef = useRef<HTMLDivElement>(null);
+
+  const handleButtonClick = (index: number) => {
+    setSelected(index);
+  };
+
+  useEffect(() => {
+    if (menuRef.current) {
+      setMenuHeight(menuRef.current.clientHeight);
+    }
+  }, []);
+
+  return (
+    <div className="mx-auto max-w-7xl">
+      <h3 className="text-3xl font-medium text-black dark:text-white">Departamentos</h3>
+
+      <motion.div
+        variants={{
+          hidden: {
+            opacity: 0,
+            y: -20,
+          },
+          visible: {
+            opacity: 1,
+            y: 0,
+          },
+        }}
+        initial="hidden"
+        whileInView="visible"
+        transition={{ duration: 1, delay: 0.1 }}
+        viewport={{ once: true }}
+        style={{ padding: '3rem 2rem' }}
+        className="flex justify-center items-start h-full"
+      >
+        <div
+          ref={menuRef}
+          className="w-full flex items-start gap-8"
+        >
+          <div className="flex flex-col gap-4 w-1/4">
```

Comment:
I think this would look better if it were centered based off of the cards on the right.